### PR TITLE
OpenMandriva: support more architectures

### PR DESCRIPTION
**Pull request by tpgxyz**
This allow to build image on other than x86_64 architectures for OpenMandriva

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -2191,6 +2192,8 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     if not do_run_build_script and args.bootable:
         packages |= {"kernel-release-server", "binutils", "systemd-boot", "dracut", "timezone", "systemd-cryptsetup"}
         configure_dracut(args, root)
+    if args.network_veth:
+        packages |= {"systemd-networkd"}
```
Comment (behrmann) [inline mkosi/__init__.py:2196]: Could you split this into a separate commit or at least update the commit message, since this is unrelated to the architectures?

**Review by behrmann — COMMENTED** [review]
LGTM

File: mkosi/__init__.py (hunk 1 above)
Comment (tpgxyz) [reply mkosi/__init__.py:2196]: Well i forgot to mention in commit about this
